arch/risc-v/src/mpfs/Kconfig

```diff
@@ -46,6 +46,9 @@ config MPFS_BOOTLOADER
 	---help---
 		This NuttX image is used as a bootloader, which will boot only on one hart, putting the others in WFI
 
+config N_IRQ_STACKS
```

Comment:
it's wrong to define N_IRQ_STACKS in here and sched/Kconfig?

Reply:
No, this is actually a legal way to override the default value in another Kconfig file (board specific), https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt:
"
A config option can be defined multiple times with the same
name, but every definition can have only a single input prompt and the
type must not conflict.
"
and
"
Default values are not limited to the menu entry where they are
  defined. This means the default can be defined somewhere else or be
  overridden by an earlier definition
"